akka-sample-spring-extension-java/src/main/java/org/example/App.java

```diff
@@ -0,0 +1,29 @@
+package org.example;
+
+import org.example.akka.di.SpringExtension;
+import org.example.akka.message.Message;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ConfigurableApplicationContext;
+
+import akka.actor.ActorRef;
+import akka.actor.ActorSystem;
+
+@SpringBootApplication
+public class App {
+
+	public static void main(String[] args) {
+		ConfigurableApplicationContext context = SpringApplication.run(App.class, args);
```

Comment:
Please don't format the code using tabs -- follow the same style as other examples do, which is 2 spaces